src/app/app-routing.module.ts

```diff
@@ -14,12 +14,13 @@ const routes: Routes = [
   { path: "agent/login", component: AgentLoginComponent },
   { path: "agent/dashboard", component: AgentDashboardComponent },
   { path: "counter/login", component: CounterLoginComponent },
-  { path: 'traveler/dashboard', component: TravelerComponent },
-  { path: 'traveler/login', component: TravelerLoginComponent },
+  { path: "traveler/dashboard", component: TravelerComponent },
+  { path: "traveler/login", component: TravelerLoginComponent },
   {
     path: "counter",
     component: CounterComponent,
     children: [
+      { path: "", pathMatch: "full", redirectTo: "traveler" },
```

Comment:
Is the result of this that any url with counter/* will be brought to the counter or any url utopia.com/* will be brought counter

Reply:
Neither. The result is that the URI `counter` redirects to `counter/traveler`. I don't have anything for page-not-found yet.